README.md

```diff
@@ -1,7 +1,7 @@
 # gDR
 Umbrella package for the gDR programmatic R interface
 
-The gDR suite offers a full stack solution for storing, processing, and visualizing drug response data. This enables a range of users across computational savvy, (i.e. lab scientists and computational scientists alike) to access the same, standardized data. The suite is made up of several core R packages, a database, a shiny visualization app. 
+The gDR suite offers a full stack solution for storing, processing, and visualizing drug response data. This enables a range of users across computational savvy, (i.e. lab scientists and computational scientists alike) to access the same, standardized data. The suite is made up of several core R packages, a database, and a shiny visualization app. 
 
 
 ```       
```

Comment:
Has the idea of ​​doing this graph with `mermaid` been considered?

Reply:
Just looked it up. Worth looking into if this diagram gets more complex.